src/main/java/org/jabref/gui/util/ViewModelListCellFactory.java

```diff
@@ -203,6 +215,9 @@ protected void updateItem(T item, boolean empty) {
                         Subscription subscription = BindingsHelper.includePseudoClassWhen(this, pseudoClassWithCondition.getKey(), condition);
                         subscriptions.add(subscription);
                     }
+                    if ((validationStatusProperty != null) && validationStatusProperty.call(viewModel).getHighestMessage().isPresent()) {
+                        setStyle("-fx-background-color: -jr-warn");
```

Comment:
I think adding a pseudo-class `invalid` instead of the style is more flexible in the long term (as it is a clearer separation between ui and ui-logic)